components.yml

```diff
@@ -10,20 +10,23 @@ deployed_component_versions:
   # e.g.
   # ingest: v0.1.2
   upload: v2.2.2
-  metadata-schema: 4.4.0
+  metadata-schema: http://api.ingest.integration.data.humancellatlas.org/schemas/search/latestSchemas
```

Comment:
Wondering whether this should point to `api.ingest.staging` instead of `api.ingest.integration`?

Reply:
fixed :)